docs/additional-deployment-options/configure_cloudwatch_logs_delivery.md

```diff
@@ -0,0 +1,39 @@
+---
+id: cloudwatch-logs-delivery
+title: Configure Cloudwatch Logs Delivery
+hide_title: true
+---
+
+# Configure Cloudwatch Logs Delivery
+
+As an optional configuration step, it is possible to deliver CloudWatch API execution and access logs to a cross-account shared AWS::Logs::Destination. An operator does this by configuring the `cumulus` module for [your deployment](../deployment/README.md#configure-and-deploy-the-cumulus-tf-root-module) as shown below. The value of the `log_destination_arn` variable is the ARN of a writeable log destination.
+
+The value can be either an [AWS::Logs::Destination](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-logs-destination.html) or a [Kinesis Stream](https://aws.amazon.com/kinesis/data-streams/) ARN to which your account can write.
+
+```hcl
+log_api_gateway_to_cloudwatch = true
+log_destination_arn           = arn:aws:[kinesis|logs]:us-east-1:123456789012:[streamName|destination:logDestinationName]
+```
+
+## Logs Sent
+
+Be default, the following logs will be sent to the destination when one is given.
+
+* Ingest logs
```

Comment:
I thought you were going to call out the changes from terraform.tfvars here?
these:  Somewhere in this PR i think you need to mention to configure these values.

metrics_es_host = "d1p5x1tmuqetix.cloudfront.net"
metrics_es_username = "user"
metrics_es_password = "password"

Reply:
Added it to the "ESDIS Metrics" section